ext/opentelemetry-ext-pymysql/tests/test_pymysql_integration.py

```diff
@@ -17,22 +17,61 @@
 import pymysql
 
 import opentelemetry.ext.pymysql
-from opentelemetry.ext.pymysql import trace_integration
+from opentelemetry.ext.pymysql import PymysqlInstrumentor
+from opentelemetry.sdk import resources
 from opentelemetry.test.test_base import TestBase
 
 
 class TestPyMysqlIntegration(TestBase):
-    def test_trace_integration(self):
-        with mock.patch("pymysql.connect"):
-            trace_integration()
-            cnx = pymysql.connect(database="test")
-            cursor = cnx.cursor()
-            query = "SELECT * FROM test"
-            cursor.execute(query)
+    def tearDown(self):
+        super().tearDown()
+        # ensure that it's uninstrumented if some of the tests fail
```

Comment:
```suggestion
```
Nit: this is called also if the test pass

Reply:
Done.